packages/realm/src/binding.ts

```diff
@@ -16,13 +16,13 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-import { IndexSet, Int64, ObjKey, Timestamp } from "../generated/ts/native.mjs";
+import { IndexSet, Int64, ObjKey, Timestamp } from "realm/binding";
 
 /** @internal */
-export * from "../generated/ts/native.mjs"; // core types are transitively exported.
+export * from "realm/binding";
```

Comment:
I know we just spend a long time figuring out how to make the re-export of every type from `core.ts` work through `native.d.mts`, but what if we just ... didn't. Do you think it would actually be cleaner to have this file do the `export * from "../generated/ts/core";` rather than doing that in the wrapper and the .d.ts? That might let you remove the virtual `realm/binding/core` path.

Feel free to ignore if you are just tired of fucking around with that file 😁